.legacy/requirements.txt

```diff
@@ -1,5 +1,8 @@
-# This file needs to be updated everytime the Pipenv files are.
-colorama>=0.4.1
-appdirs>=1.4.4
-readchar>=2.0.1
-blessed>=1.17.0
\ No newline at end of file
+-i https://pypi.org/simple
+appdirs==1.4.4
+blessed==1.17.11
+colorama==0.4.4
+numpy==1.19.2
+readchar==2.0.1
+six==1.15.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
```

Comment:
The python version are wrong: we are using f-strings extensively so we are only compatible with python 3.6+ as described in the setup.py file.